composer.json

```diff
@@ -24,15 +24,14 @@
         "symfony/console": "^5.1.8",
         "symfony/event-dispatcher-contracts": "^2.2",
         "yiisoft/di": "@dev",
-        "yiisoft/friendly-exception": "^1.0",
-        "yiisoft/injector": "^1.0",
-        "yiisoft/yii-event": "^3.0@dev"
```

Comment:
If you remove `yii-event` from where we will get the event dispatcher configuration, since this is `yii-console`, you should use `yiisoft/event-distpacher`

Reply:
No event dispatcher is used in this package, so we musn't add any implementation. I'll add `yiisoft/event-dispatcher` into the demo and app template repos. This is discussed with @samdark and @yiiliveext in the core team chat.